Prototope/Layer+Layout.swift

```diff
@@ -66,6 +66,30 @@ extension Layer {
 	}
 	
 	
+	/** Moves the receiver so that its left side is aligned with the left side of its parent layer. */
+	public func moveToLeftSideOfParentLayer(margin: Double = 0.0) {
+		if let parent = self.parent {
+			self.frameMinX = -margin
```

Comment:
Aha. `moveToLeftOfSiblingLayer` is different because it’s with respect to the given `siblingLayer`, whereas `moveToLeftSideOfParentLayer` moves with respect to the `parent` layer.

If you have a layer called `parent`, with child layers `child1` and `child2`, `child1.moveToLeftSideOfParentLayer()` will move to be aligned with parent layer’s origin (you can think of this as “stick child to the left side of its parent” where `child1.moveToLeftOfSiblingLayer(child2)` will place `child1` beside `child2` on the left side.

I’ll grant it’s a bit confusing at first glance and I’m probably blinded by being too used to them :smile: but they’re pretty nice once you get the swing of them.